features/facebookImport/src/views/explore/details.jsx

```diff
@@ -6,10 +6,11 @@ import "./details.css";
 const ExploreDetails = () => {
     const history = useHistory();
     const { activeAnalysis } = history.location.state;
+    activeAnalysis.mode = "DETAILS";
```

Comment:
This is apparently an enum. Have you considered defining the different modes in `activeAnalysis` and then doing something like:

```js
activeAnalysis.mode = activeAnalysis.DETAILS
```

Even better,

```
activeAnalysis.setDetailsMode()
```

This way it's not really respecting encapsulation, client is using a specific implementation, and this does not constrain the values you'll be using. Valid as a first approximation, but complicated for any future refactoring.

Reply:
Wouldn't this be `Story.MODE.DETAILS`?

Reply:
You right, of course